ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -231,34 +216,35 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     ///   - containerWidth: The width of the cell's super view (e.g. the table view's width)
     ///   - isInSelectionMode: Boolean describing if the cell is in multi-selection mode which shows/hides a checkmark image on the leading edge
     /// - Returns: a value representing the calculated height of the cell
-    @objc public class func height(tokens: TableViewCellTokens = .init(),
-                                   title: String,
-                                   subtitle: String = "",
-                                   footer: String = "",
-                                   attributedTitle: NSAttributedString? = nil,
-                                   attributedSubtitle: NSAttributedString? = nil,
-                                   attributedFooter: NSAttributedString? = nil,
-                                   isAttributedTitleSet: Bool = false,
-                                   isAttributedSubtitleSet: Bool = false,
-                                   isAttributedFooterSet: Bool = false,
-                                   titleFont: UIFont? = nil,
-                                   subtitleFont: UIFont? = nil,
-                                   footerFont: UIFont? = nil,
-                                   titleLeadingAccessoryView: UIView? = nil,
-                                   titleTrailingAccessoryView: UIView? = nil,
-                                   subtitleLeadingAccessoryView: UIView? = nil,
-                                   subtitleTrailingAccessoryView: UIView? = nil,
-                                   footerLeadingAccessoryView: UIView? = nil,
-                                   footerTrailingAccessoryView: UIView? = nil,
-                                   customViewSize: MSFTableViewCellCustomViewSize = .default,
-                                   customAccessoryView: UIView? = nil,
-                                   accessoryType: TableViewCellAccessoryType = .none,
-                                   titleNumberOfLines: Int = 1,
-                                   subtitleNumberOfLines: Int = 1,
-                                   footerNumberOfLines: Int = 1,
-                                   customAccessoryViewExtendsToEdge: Bool = false,
-                                   containerWidth: CGFloat = .greatestFiniteMagnitude,
-                                   isInSelectionMode: Bool = false) -> CGFloat {
+    public class func height(tokenSet: TableViewCellTokenSet? = nil,
```

Comment:
Is this branch up to date with all the latest TVC changes that had been going in to main?

Reply:
Almost certainly not; it's likely not even merged with changes in `fluent2-tokens`, since it was spun off before those changes began. I'll spin up a second iteration that at least pulls that all in.

Reply:
Checked: it is caught up with `fluent2-tokens` but not `main`. I'll wait for that merge to complete before addressing any needed changes.